ui/plugin-lifecycle/src/main/interfaces/Tenant.ts

```diff
@@ -0,0 +1,40 @@
+export interface Tenant {
```

Comment:
It seems like this may duplicate the available Organization models that are already part of @vcd/bindings?

Reply:
I can't find this interface are you sure this is the name of it?

Reply:
`export class QueryResultOrgRecordType extends QueryResultRecordType {
    public canPublishCatalogs?: boolean;
    public deployedVMQuota?: number;
    public displayName?: string;
    public isEnabled?: boolean;
    public isReadOnly?: boolean;
    public name?: string;
    public numberOfCatalogs?: number;
    public numberOfDisks?: number;
    public numberOfGroups?: number;
    public numberOfVApps?: number;
    public numberOfVdcs?: number;
    public storedVMQuota?: number;
}`